modules/dgi_ir_spreadsheet_ingest_example/dgi_ir_spreadsheet_ingest_example.info.yml

```diff
@@ -0,0 +1,8 @@
+name: DGI IR Spreadsheet Ingest Example
+description: Provides a base CSV migration template and script for creating Research Output nodes, based on the out-of-the-box Research Output implementation.
+type: module
+package: DGI
+core: 8.x
+dependencies:
+  - islandora_institutional_repository
+  - islandora_spreadsheet_ingest
```

Comment:
... `public` vs `private` stuff... strange to have a public repo pointing at a private requirement.

Reply:
This is still a thing...